libdrgn/arch_s390x.c

```diff
@@ -249,7 +466,10 @@ const struct drgn_architecture_info arch_info_s390x = {
 	.linux_kernel_get_initial_registers =
 		linux_kernel_get_initial_registers_s390x,
 	.apply_elf_reloc = apply_elf_reloc_s390,
-
+	.linux_kernel_pgtable_iterator_create =  linux_kernel_pgtable_iterator_create_s390,
+	.linux_kernel_pgtable_iterator_destroy = linux_kernel_pgtable_iterator_destroy_s390,
+	.linux_kernel_pgtable_iterator_init = linux_kernel_pgtable_iterator_init_s390,
+	.linux_kernel_pgtable_iterator_next = linux_kernel_pgtable_iterator_next_s390,
```

Comment:
Can we name these all `_s390x` for consistency with the other kernel callbacks?

```suggestion
	.linux_kernel_pgtable_iterator_create =  linux_kernel_pgtable_iterator_create_s390x,
	.linux_kernel_pgtable_iterator_destroy = linux_kernel_pgtable_iterator_destroy_s390x,
	.linux_kernel_pgtable_iterator_init = linux_kernel_pgtable_iterator_init_s390x,
	.linux_kernel_pgtable_iterator_next = linux_kernel_pgtable_iterator_next_s390x,
```